app/lib/service/email/backend.dart

```diff
@@ -116,6 +122,13 @@ class EmailBackend {
           entry.bodyText!,
         ));
         sent.add(recipientEmail);
+      } on EmailSenderException catch (e, st) {
+        _logger.warning('Email sending failed (claimId="$claimId").', e, st);
+        // Resetting attempt count when the failure seems to be infrastructure-related.
+        if (e.status >= 500) {
+          resetAttempt = true;
+          break;
+        }
```

Comment:
Shouldn't we limit number of retries anyway? if there is consistent infrastructure issues, we want retries to exhaust.

Reply:
We do limit the retries, however, that limit is applied on the single message send operation. If the backend wants to send the mail to 5 recipients, we have separate retries, and until this PR, we have tried to send them separately. The change here would be that after the first authentication-related failure we will not try to send the rest of the recipients, and also reset the attempt count on the datastore entity, so that we don't penalize the recipients just because we were not able to get an auth token.

Reply:
If you reset the attempts count, then this can stay around forever, correct?

Reply:
Good point, I agree. I can either remove, or introduce a small rule that limits the impact, e.g. reset only happens in the first few hours, after that we are bound by the attempt limits. I guess we can just remove now, and if there are issues, we introduce the reset with a time-limited rule.

Reply:
Let's remove `resetAttempt` and then ship it